pkgs/development/python-modules/tree-sitter/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "tree-sitter";
+  version = "0.20.0";
+
+  src = fetchPypi {
+    pname = "tree_sitter";
+    inherit version;
+    sha256 = "sha256-GUD2S+HoycPA40oiWPHkwyQgdTTVse78WrKWCp2Y9mg=";
+  };
+
+  meta = with lib;{
+    homepage = "https://pypi.org/project/tree-sitter";
```

I would be better to specify what's provided on the PyPI package page.
```suggestion
    homepage = "https://github.com/tree-sitter/py-tree-sitter";
```

---

Thanks for the suggestion, applied.